app/components/message_attachments/attachment_author.tsx

```diff
@@ -63,7 +63,9 @@ export default class AttachmentAuthor extends PureComponent<Props> {
                 <FastImage
                     source={{uri: icon}}
                     key='author_icon'
+                    id='author_icon'
                     style={style.icon}
+                    renderErrorImage={true}
```

Comment:
errorImageSize = 12 ?

Reply:
Wondering what will happen to the extra styles applied, I did not see any of that with the broken image in the RetryFastImage component, see the style for this one, is applying a marginRight as well